onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -570,7 +570,7 @@ contract CartesiDAppTest is TestBase {
         assertEq(address(simpleContract).balance, transferAmount);
     }
 
-    function testWithdrawEtherToContractWithPayload(
+    function testWithdrawEtherToContractWithRandomPayload(
```

Comment:
When I read "random" payload, my first thought is that it is a fuzzy variable.
Instead, it is a hard-coded, seemingly random payload.
So, we could say it is just a wrong payload.

Reply:
Make sense 👍